acceptance_tests/__init__.py

```diff
@@ -3,6 +3,6 @@
 from splinter import Browser
 
 if os.getenv('HEADLESS', 'True') == 'True':
-    browser = Browser('phantomjs')
+    browser = Browser('chrome', headless=True)
```

Comment:
We don't need to set headless=True here again. Can you also make sure that this will also work in Jenkins?

Reply:
We need to set headless here to trigger the headless version of chrome I think?
I'm looking into how to make sure we have chromedriver in Jenkins today :+1:

Reply:
I added installing chromedriver to the jenkinsfile so hopefully that works